src/frontend/komponenter/Fagsak/Vedtak/TrekkILøpendeUtbetalingNy/TrekkILøpendeUtbetaling.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { AddCircle } from '@navikt/ds-icons';
+import { Button, Heading, Table } from '@navikt/ds-react';
+
+import type { IRestTrekkILøpendeUtbetaling } from '../../../../typer/eøs-trekk-i-løpende-ytelse';
+import NyFeilutbetaltValutaPeriode from './NyFeilutbetaltValutaPeriode';
+import TrekkILøpendeUtbetalingListeElement from './TrekkILøpendeUtbetalingListeElement';
+
+interface ITrekkILøpendeUtbetaling {
+    behandlingId: number;
+    trekkILøpendeUtbetalingListe: IRestTrekkILøpendeUtbetaling[];
+    settErUlagretNyTrekkILøpendeUtbetaling: (erUlagretNyTrekkILøpendeUtbetaling: boolean) => void;
+    erLeservisning: boolean;
+    settVisTrekkILøpendeUtbetaling: (visTrekkILøpendeUtbetaling: boolean) => void;
+}
+
+const FlexColumnDiv = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin-bottom: 3.5rem;
+    margin-top: 2rem;
+`;
+
+const FlexRowDiv = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+
+const TrekkILøpendeUtbetaling: React.FC<ITrekkILøpendeUtbetaling> = ({
+    trekkILøpendeUtbetalingListe,
+    settErUlagretNyTrekkILøpendeUtbetaling,
+    erLeservisning,
+    settVisTrekkILøpendeUtbetaling,
+    behandlingId,
+}) => {
+    const [ønskerÅLeggeTilNyPeriode, settØnskerÅLeggeTilNyPeriode] = useState(
+        !trekkILøpendeUtbetalingListe
+    );
+
+    useEffect(() => {
+        settErUlagretNyTrekkILøpendeUtbetaling(ønskerÅLeggeTilNyPeriode);
+    }, [ønskerÅLeggeTilNyPeriode]);
+
+    if (trekkILøpendeUtbetalingListe.length === 0 && !ønskerÅLeggeTilNyPeriode) {
+        settVisTrekkILøpendeUtbetaling(false);
```

Comment:
Hva skjer her? Er det at vi skjuler hele seksjonen hvis man avbryter en ny periode? Eller?

Reply:
Ja skjuler hele komponenten som viser overskrift + tabell + knapp for å legge til ny. Ref dette ønsket på favro-kortet: 
<img width="770" alt="image" src="https://user-images.githubusercontent.com/46678893/207885379-9fab39cb-8652-4fc2-b275-e78b1231f1d4.png">

Reply:
I så fall: 
- typing av prop `skjulTrekkILøpendeUtbetaling: () => void;` 
- send inn prop `skjulTrekkILøpendeUtbetaling={() => settVisTrekkILøpendeUtbetaling(false)}`
- kall på `skjulTrekkILøpendeUtbetaling()`

evt `lukkTrekkI...`